aws/resource_aws_emr_cluster.go

```diff
@@ -1139,8 +1139,21 @@ func resourceAwsEMRClusterRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	var stepSummaries []*emr.StepSummary
+	states := []string{
+		"PENDING",
+		"CANCELLED",
+		"FAILED",
+		"CANCEL_PENDING",
+		"RUNNING",
+		"INTERRUPTED",
+	}
+	var stepStates []*string
+	for _, v := range states {
+		stepStates = append(stepStates, &v)
+	}
```

Comment:
Since `v` will be created only once when loop starts, it will always have the same pointer value. In order to avoid that, use reference to the element of the array via index. More info: https://medium.com/codezillas/uh-ohs-in-go-slice-of-pointers-c0a30669feee

```suggestion
	for i := range states {
		stepStates = append(stepStates, &states[i])
	}
```